scripts/utils/price_utils.js

```diff
@@ -208,16 +174,14 @@ const isPriceReasonable = async (baseTokenData, quoteTokenData, price, acceptedP
     console.log(
       `Warning: the chosen price differs by more than ${acceptedPriceDeviationInPercentage} percent from the price found on ${onlinePriceSlice.source}.`
     )
-    console.log(`         chosen price: ${price} ${quoteTokenData.symbol} bought for 1 ${baseTokenData.symbol}`)
-    console.log(
-      `         ${onlinePriceSlice.source} price: ${onlinePrice} ${quoteTokenData.symbol} bought for 1 ${baseTokenData.symbol}`
-    )
+    console.log(`    chosen price: ${price} ${buyToken.symbol} bought for 1 ${sellToken.symbol}`)
+    console.log(`    ${onlinePriceSlice.source} price: ${onlinePrice} ${buyToken.symbol} bought for 1 ${sellToken.symbol}`)
     return false
   }
   return true
 }
 
-const checkNoProfitableOffer = async (order, exchange, tokenInfo, globalPriceStorage = null) => {
+const checkNoProfitableOffer = async (order, tokenInfo, globalPriceStorage = null) => {
   const currentMarketPriceSlice = await getOneinchPrice(
     await tokenInfo[order.buyToken],
```

Comment:
Why doesn't it have to be changed here?

Reply:
Because exchange was not being used. This is just an improvement (removing an unused variable)

Reply:
I will revert this change as well and make a new PR then

Reply:
thanks for removing the "exchange" thing out of this PR. But actually, I was talking about why the order of quote and base token is not changed here.

Reply:
I looked into it and I am quite convinced that it needs to be changed.